src/definitions/modules/modal.js

```diff
@@ -110,6 +110,10 @@ $.fn.modal = function(parameters) {
           }
           if(module.has.configActions()){
             var $actions = $module.find(selector.actions).addClass(settings.classActions);
+            if ($actions.length === 0) {
+              $actions = $('<div/>', {class: className.actions + ' ' + (settings.classActions || '')}).appendTo($module);
+            }
+            $actions.empty();
```

Comment:
Nit) I think empty can be skipped when the action container is created dynamically.

```suggestion
            } else {
              $actions.empty();
            }
```